apps/webapp/app/send/send-form.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import React, { useState } from 'react';
+import { InputBlock, InputToken, ResponsiveImage } from '../../shared';
+import { Button, Switch } from 'ui';
+import { Asset } from './types';
+import { assets } from './constants';
+
+export const SendForm = () => {
+  const [asset, setAsset] = useState<Asset>(assets[0]!);
+  const [recepient, setRecepient] = useState('');
+  const [memo, setMemo] = useState('');
+  const [amount, setAmount] = useState('');
+  const [hidden, setHidden] = useState(false);
+
+  return (
+    <form
+      className='flex flex-col gap-2'
+      onSubmit={e => {
+        e.preventDefault();
+      }}
+    >
+      <InputBlock
+        label='Recepient'
+        placeholder='Enter the address'
+        className='mb-1'
+        value={recepient}
+        onChange={e => setRecepient(e.target.value)}
+      />
+      <InputToken
+        label='Amount to send'
+        placeholder='Enter an amount'
+        className='mb-1'
+        asset={asset}
+        setAsset={setAsset}
+        value={amount}
+        onChange={e => setAmount(e.target.value)}
+      />
```

Comment:
At the moment, this is allowing letters as input. Think we need to make use of input's

```
<input type="number">
```

if we can to help with field validation